metrics/deploy/lib/cli/git/git_command.dart

```diff
@@ -2,7 +2,13 @@ import 'package:process_run/process_run.dart' as cmd;
 
 /// class wrapping up gcloud CLI
 class GitCommand {
+  /// Git clone command.
```

Comment:
Would it be more clear?
```suggestion
  /// Clones the git repository from the given [repoURL] to the given [srcPath].
```